business_requirement/models/business.py

```diff
@@ -453,3 +468,14 @@ class Project(models.Model):
     @api.depends('br_ids')
     def _br_count(self):
         self.br_count = len(self.br_ids)
+
+
+class Task(models.Model):
+    _inherit = 'project.task'
+
+    """ Link the task and the business requirement """
+    business_requirement_id = fields.Many2one(
```

Comment:
@duanyp1991 actually works that way I pass the context and it's filled. Maybe should I get context here and not in the view? 
cc @duanyp1991